backend/ibmcos/ibmSigner.go

```diff
@@ -0,0 +1,33 @@
+package ibmcos
```

Comment:
Can you put this into `backend/s3` as `ibm_signer.go` please. I don't think it needs to be its own package.

Reply:
done